src/App.js

```diff
@@ -28,8 +39,8 @@ function App () {
         />
         <Route path="/money-transactions" element={
         <>
-          <MoneyTransactionCreate />
-          <MoneyTransactionList />
+          <MoneyTransactionCreate users={users} setMoneyTransactions={setMoneyTransactions}/>
+          <MoneyTransactionList moneyTransactions={moneyTransactions} users={users}/>
```

Comment:
Move these two components into there own page components (a parent component called e.g. MoneyTransactionPage) and fetch the users/transactions from within this component. In the current state the data does not load when you navigate directly to money-transaction without going through "/" 

-1 Punkte, sonst passt es

Reply:
hätts gerade probiert, immer wenn ich die zwei Komponenten in eine neue Komponente "MoneyTransactionPage" packe, dann ist die App kaputt 😞
<img width="789" alt="Bildschirmfoto 2022-03-31 um 16 30 53" src="https://user-images.githubusercontent.com/85451884/161079784-020d8c0e-3c33-47b3-8316-7738d86e8cfe.png">